### ISSUE TITLE
I want to load a local custom spacy model in python script to run nlp. I havent found a way to access the model folder in python. It keeps returning model not found error

### ISSUE THREAD

**Issue by jaggadas** [chaquo/chaquopy#638]

**Comment by mhsmith** [comment]
This is answered in [the FAQ](https://chaquo.com/chaquopy/doc/current/faq.html#faq-read).

**Comment by jaggadas** [comment]
> This is answered in [the FAQ](https://chaquo.com/chaquopy/doc/current/faq.html#faq-read).

I actually wanted to open a model folder, not a file, is there anyway to do that

**Comment by mhsmith** [comment]
Yes, the `__file__` and `os.environ["HOME"]` techniques will both work in the same way with a folder.

**Comment by jaggadas** [comment]
I solved the issue ! thanks
